.github/workflows/testing.yml

```diff
@@ -0,0 +1,31 @@
+name: Django Testing
+
+on:
+    push:
+        branches: [ "main", "dev" ]
+    pull_request:
+        branches: [ "main", "dev" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    defaults: 
+        run:
+            working-directory: ./evergreen
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.12
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.12
+    - name: Install environment
+      run: |
+        python -m pip install --upgrade pip
+        pip install poetry
+        poetry install
+
+    - name: Django Testing
+      run: |
+        poetry run manage.py test
```

Comment:
This should be `poetry run ./manage.py test`, though that now fails for a different reason for me. Looks like a Django config problem now I think? Should compare the settings.py as of #12 with current to see what's different

Reply:
Yes the test for manage.py keeps failing for me and Im not quite sure why. The errors are in the manage.py code so.....